.github/workflows/2-feladat.yml

```diff
@@ -10,3 +10,18 @@ jobs:
       - uses: actions/checkout@v2
       - run: g++ -std=c++17 *.cpp
       - run: ./run-test.sh
+      - name: Install tools
+        run: sudo apt-get install -y valgrind cppcheck
+      - name: Gpp Error
+        run: g++ -Wall -Werror *.cpp
+      - name: Valgrind Error
+        run: valgrind --error-exitcode=1 ./a.out kakarott.json makarott.json
```

Comment:
majdnem jó, de még kell egy `--leak-check=full` is, különben nem minden hibát tekint errornak a valgrind